joplin/base/wagtail_hooks.py

```diff
@@ -79,26 +79,33 @@ def joplin_page_listing_more_buttons(page, page_perms, is_parent=False):
             attrs={'title': _("Copy page '{title}'").format(title=page.get_admin_display_title())},
             priority=20
         )
-    if page_perms.can_delete():
-        yield Button(
-            _('Delete'),
-            reverse('wagtailadmin_pages:delete', args=[page.id]),
-            attrs={'title': _("Delete page '{title}'").format(title=page.get_admin_display_title())},
-            priority=30
-        )
+    # if not page.live:
+    #     yield Button(
```

Comment:
I'm not sure why we're using the hook except for the fact that wagtail was already using it and now we're overriding it (I think, I may not have a full understanding of it).

As far as `register_page_listing_buttons` vs `register_page_listing_more_buttons` goes, the more buttons are the buttons that go in the "more" dropdown, and are registered via the hook in the more button https://github.com/cityofaustin/joplin/blob/56861d633da9b10ed4c6ce384951f42d370e7bcd/joplin/base/wagtail_hooks.py#L64

Reply:
oh, I wasn't clear with my question. I was confused how the built-in hooks relate to our custom hooks since I think both run for the given tags. So how does register_page_listing_buttons relate to our custom hook register_joplin_page_listing_buttons hook -- since I think both hooks are executed in tags that populate the 'wagtailadmin/pages/listing/_buttons.html' partial. And both set of hooks return `return {'page': page, 'buttons': buttons}`. So I'm curious the flow of tags and how order is determined, and how one tag overwrites the other tag.

FWIW, we don't have to use hooks to populate tags to be sent to partials too.